tst/rref/is_rref.tst

```diff
@@ -0,0 +1,4 @@
+gap> ReadPackage("GaussPar", "gap/RREF.g");;
```

Comment:
We assume that "GaussPar" is loaded when we run tests. So you don't need this line here. Instead you need to put it into `read.g`.

Reply:
Do we want to load this every time that we load the package? Where exactly are we going to use the function?

Reply:
Yes, that's a function that _might_ also be useful in general. So I see no reason to make it an "internal function" via the `GAUSS_` prefix. We will need this function in #54.